hfs/selectors/lazyHierarchicalFeatureSelector.py

```diff
@@ -10,7 +10,7 @@
 from hfs.selectors import HierarchicalEstimator
 
 
-class LazyHierarchicalFeatureSelector(HierarchicalEstimator, ABC):
+class LazyHierarchicalFeatureSelector(ABC, HierarchicalEstimator):
     """
     Abstract class used for all lazy hierarchical feature selection methods.
 
```

Comment:
While looking into `ABC` and how it is used, I realised that the doc string here states that eveery `method` (should be `subclass`) should implement `select_and_predict` but the actual method `select_and_predict()` here does not have the `@abstractmethod` decorator. So if a subclass doesn't implement this method, it wouldn't be caught.
Could you please update that even though it is not technically part of this PR so we don't miss it later?

Reply:
Added both import and decorator of `select_and_predict`.